diff --git a/app/build.gradle.kts b/app/build.gradle.kts
index be3711a9..1fcf95ce 100644
--- a/app/build.gradle.kts
+++ b/app/build.gradle.kts
@@ -158,7 +158,7 @@ android {
     productFlavors {
         create("full") {
             dimension = "distribution"
-            applicationIdSuffix = ".livetv"
+            applicationIdSuffix = ".jellyfintv"
             buildConfigField("boolean", "FEATURE_PLUGINS_ENABLED", "true")
             buildConfigField("boolean", "FEATURE_IN_APP_UPDATES_ENABLED", "true")
             buildConfigField("boolean", "FEATURE_IN_APP_TRAILERS_ENABLED", "true")
