diff --git a/app/build.gradle.kts b/app/build.gradle.kts
index aa7c8534..d58b4cae 100644
--- a/app/build.gradle.kts
+++ b/app/build.gradle.kts
@@ -276,8 +276,8 @@ android {
         abi {
             isEnable = !buildingAppBundle
             reset()
-            include("armeabi-v7a", "arm64-v8a", "x86", "x86_64")
-            isUniversalApk = true
+            include("armeabi-v7a") // fork: the workflow publishes only the v7a APK - skip building the other ABIs + universal
+            isUniversalApk = false
         }
     }
 
